### DISCUSSION TITLE
Install on replit.com

### DISCUSSION THREAD

**Discussion by ilyas013** [FabCx/TradingView-Webhook-Bot#104] [Q&A]
Is it possible, install and run on replit.com?

**Comment by fabston** [comment]
Hi, I never tried that but I don’t think it’ll work since it relies on flask.

**Comment by mamolas** [comment]
Hey @fabston  As per the following guide it's possible and quick to deploy on replit:
[https://www.tradingview.com/chart/ETHUSD/uQCb82ML-How-to-create-simple-web-hook-to-send-alerts-to-Telegram/](https://www.tradingview.com/chart/ETHUSD/uQCb82ML-How-to-create-simple-web-hook-to-send-alerts-to-Telegram/)

Re-posted in YT as well with several thousands views [https://www.youtube.com/watch?v=otUinXXejdE](https://www.youtube.com/watch?v=otUinXXejdE)

As per the guide and the vid it should work nicely even for a newbie but unfortunately I got: "Telegram Error: > reject_bytes is on and 'BTCUSD' is bytes"  I already asked support in the guide's thread which has hundred of positive comments but it seems an update or something recent broke it, now it's mandatory to send the alert in Json format and also set a query parameter to notify that post body in Json, sounds kind of too messy to handle for the average user, even myself as a rookie coder got in trouble to make it work even sending a Json alert, but worst or impossible if you don't have the pine script source code of the indicator to format the alert properly in Json. Can you please take a look? Maybe some small fix will allow again to forward plain text alerts easily as it used to be, Thanks  

EDIT: I was able to get alerts on the telegram channel simply by adding " ?jsonRequest=true " to pass the parameter correctly at the URL but still to send plain text would be easier :+1:
